huddleupui/src/layouts/CommunityLayout.jsx

```diff
@@ -530,8 +563,107 @@ export default function CommunityLayout({ children, allowedUserTypes, canNotMemb
 									</Row>
 								</Card>
 							)}
-						</Drawer>
```

Comment:
this is community sidebar drawer closing, you shouldnt delete it

Reply:
i think it is ok now?